CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+## Thanks
+
+Thank you for considering contributing to **GrPPI. Our aim is to make **GrPPI** useful for the software comunity at large. 
+Any kind of contribution is highly appreciated.
+
+## Why should you read this guideline?
+
+Following this guideline will help you to keep contact with **GrPPI** team.
+This will result in shorter time to process your bug reports, suggestions and pull requests.
+
+## Ways of contributing to GrPPI
+
+You may contribute in a number of ways:
+
+* Filing bug reports.
+* Suggesting new features.
+* Contributing with documentation pull requests.
+* Contributing with pull requests with improved implementation of patterns or back ends.
+
+# Contributing go GrPPI
+
+## General rules
+
+* Please read and follow the GrPPI Code of Conduct.
+
+If you are contributing with documentation:
+
+* Please, provide improvements as small as possible, as pull requests.
+* If a change in documentation structure is needed, we prefer that you file a suggestion.
+
+If you are contributing with code:
+
+* Ensure that your pull request passes al unit tests for the set of supported platforms.
+* For minor issues you can submit a pull-request without filing and issue.
+* For non-trivial issue, please, file an issue that can be discussed, before a decision is taken on how to process it.
+* Software pull requests should be as small as possible and providing only one feature per pull request.
+
+## How to file a bug report
+
+If you find a bug, please create an issue, and include the following information:
+
+* Type and version of compiler that you used.
+* Operating system.
+* Processor and architecture.
+* Smallest use case that we can use to reproduce the the bug.
```

Comment:
I think that the following sentence would be better:

The smallest use case that can be used to reproduce the the bug